src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/BaseFlowPathRemovalAction.java

```diff
@@ -57,11 +57,8 @@ protected void updateIslsForFlowPath(FlowPath... paths) {
     }
 
     private void updateAvailableBandwidth(SwitchId srcSwitch, int srcPort, SwitchId dstSwitch, int dstPort) {
-        long usedBandwidth = flowPathRepository.getUsedBandwidthBetweenEndpoints(srcSwitch, srcPort,
-                dstSwitch, dstPort);
-        log.debug("Updating ISL {}_{} - {}_{} with used bandwidth {}", srcSwitch, srcPort, dstSwitch, dstPort,
-                usedBandwidth);
-        islRepository.updateAvailableBandwidth(srcSwitch, srcPort, dstSwitch, dstPort, usedBandwidth);
+        log.debug("Updating ISL {}_{} - {}_{} with used bandwidth", srcSwitch, srcPort, dstSwitch, dstPort);
```

Comment:
`with used bandwidth"` 
There is no used bandwidth in log now

Reply:
Right, the correct log message is in IslRepository. This excessive one has been removed.